pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/TableConfigsRestletResource.java

```diff
@@ -196,7 +196,12 @@ public ConfigSuccessResponse addConfig(
     }
     TableConfigs tableConfigs = tableConfigsAndUnrecognizedProps.getLeft();
     String databaseName = DatabaseUtils.extractDatabaseFromHttpHeaders(httpHeaders);
-    validateConfig(tableConfigs, databaseName, typesToSkip);
+    if (typesToSkip == null) {
+      // Skip validate task type for table creation as the schema is not yet registered.
+      validateConfig(tableConfigs, databaseName, "TASK");
+    } else {
+      validateConfig(tableConfigs, databaseName, typesToSkip);
```

Comment:
We should append ",TASK" here to `typesToSkip`?
Looks like `typesToSkip` string is not parsed properly in `validateTaskConfigs` and assumed that only one type is passed while the endpoint expects comma separated list of types.

Reply:
right, there is only full string match inside, I feel some parsing logic is wrong there.